### PR TITLE
*: update pd client dependency

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

related to https://github.com/tikv/client-go/issues/60

When someone needs to import tidb and go client at the same time, this PR should resolve to metrics conflict problem.

### CODE REVIEW

File: mockstore/mocktikv/pd.go [hunk 1]
```diff
@@ -46,6 +47,14 @@ func (c *pdClient) GetClusterID(ctx context.Context) uint64 {
 	return 1
 }
 
+func (c *pdClient) GetAllMembers(ctx context.Context) ([]*pdpb.Member, error) {
+	return nil, nil
```
Comment (cosven) [inline mockstore/mocktikv/pd.go:51]: how about just `panic` here like other unimplemented methods?

**Review by cosven — APPROVED** [review]
LGTM

File: mockstore/mocktikv/pd.go (hunk 1 above)
Comment (disksing) [reply mockstore/mocktikv/pd.go:51]: good idea.

**Comment by cosven** [comment]
LGTM

**Pull request merged**